components/cdc/src/initializer.rs

```diff
@@ -90,7 +90,9 @@ pub(crate) struct Initializer<E> {
     pub(crate) request_id: u64,
     pub(crate) checkpoint_ts: TimeStamp,
 
-    pub(crate) speed_limiter: Limiter,
+    pub(crate) scan_limiter: Limiter,
```

Comment:
How about `scan_speed_limiter` & `scan_disk_speed_limiter` ? That looks more clear to me.

Reply:
I prefer `scan_speed_limiter` (for disk I/O) and `fetch_speed_limiter` (for total fetched bytes). The reason is we already have a configuration `incremental_scan_speed_limit`, which works based on I/O.